php/php.editor/src/org/netbeans/modules/php/editor/completion/PHPCodeCompletion.java

```diff
@@ -124,6 +123,8 @@
  */
 public class PHPCodeCompletion implements CodeCompletionHandler2 {
 
+    // for unit tests
+    static PhpVersion PHP_VERSION = null;
```

Comment:
Should be likely `volatile`.

Reply:
I'll fix it if we check php version. Thanks!